### PR TITLE
Update openapi.yaml

### PR DESCRIPTION
**Pull request by vikhyat187**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -199,89 +158,71 @@ paths:
...
-    properties:
-      userId:
+    properties: 
+      flagId:
         type: string
-      flagId: 
+      flagName:
         type: string
-      enabled: 
+      deactivate:
```
Comment (dev-gv) [inline openapi.yaml:223]: how about `deactivated`?

File: openapi.yaml [hunk 2]
```diff
@@ -177,7 +136,7 @@ paths:
           
     post:
       tags:
-        - users
+        - featureFlags-User
       summary: To enable feature flag for a user
```
Comment (dev-gv) [inline openapi.yaml:140]: how do we disable a FF for a user ?

**Review by dev-gv — CHANGES_REQUESTED** [review]
Left a couple of small ones. looks okay otherwise.

File: openapi.yaml [hunk 3]
```diff
@@ -82,35 +82,16 @@ paths:
...
-        '404':
-          description: Flag not found
         
           
   /users/feature-flags/:
     get: 
       tags:
-        - users
+        - featureFlags-User
       summary: Lists all the feature flags for the user
```
[the remark this answers is not in the document]
Comment (vikhyat187) [reply openapi.yaml:95]: yes

File: openapi.yaml (hunk 2 above)
Comment (vikhyat187) [reply openapi.yaml:140]: If we want to check when it was deactivated we may require `deactivatedAt`  attribute of type `timestamp`

**Pull request merged**